crates/flux-fhir-analysis/src/conv/struct_compat.rs

```diff
@@ -239,7 +239,7 @@ impl<'genv, 'tcx> Zipper<'genv, 'tcx> {
         for (i, ensures) in a.ensures.iter().enumerate() {
             if let rty::Ensures::Type(path, ty_a) = ensures {
                 let loc = path.to_loc().unwrap();
-                let ty_b = self.locs.get(&loc).unwrap().clone();
+                let ty_b = self.locs.get(&loc).unwrap().clone().shift_in_escaping(1);
```

Comment:
@ranjitjhala you can remove the `.clone()`

```suggestion
                let ty_b = self.locs.get(&loc).unwrap().shift_in_escaping(1);
```